wire/test/doc.go

```diff
@@ -17,5 +17,5 @@
 
 // Package test contains implementations of peer interfaces that are useful for
 // testing and test helper functions for running generic tests for different
-// wire protocol implementations.
+// types implementing the binary encoder/decoder interfaces.
```

Comment:
Because, the generic tests helpers have been moved from `wire/test` to `wire/perunio/test`. 

The package doc comment still seems to be slightly incorrect. I have updated it in 2675a903.